packages/i18n/src/es.ts

```diff
@@ -0,0 +1,65 @@
+    endsWith:       (issue) => `Final inválido: Se esperaba ${issue.expected} pero se recibió ${issue.received}`,
+    excludes:       (issue) => `Contenido inválido: Se esperaba ${issue.expected} pero se recibió ${issue.received}`,
+    finite:         (issue) => `Finito inválido: Se recibió ${issue.received}`,
+    hash:           (issue) => `Hash inválido: Se recibió ${issue.received}`,
+    hexColor:       (issue) => `Color hexadecimal inválido: Se recibió ${issue.received}`,
+    hexadecimal:    (issue) => `Hexadecimal inválido: Se recibió ${issue.received}`,
+    imei:           (issue) => `IMEI inválido: Se recibió ${issue.received}`,
+    includes:       (issue) => `Contenido inválido: Se esperaba ${issue.expected} pero se recibió ${issue.received}`,
+    integer:        (issue) => `Entero inválido: Se recibió ${issue.received}`,
+    ip:             (issue) => `IP inválida: Se recibió ${issue.received}`,
+    ipv4:           (issue) => `IPv4 inválido: Se recibió ${issue.received}`,
+    ipv6:           (issue) => `IPv6 inválido: Se recibió ${issue.received}`,
+    isoDate:        (issue) => `Fecha inválida: Se recibió ${issue.received}`,
+    isoDateTime:    (issue) => `Fecha-hora inválida: Se recibió ${issue.received}`,
+    isoTime:        (issue) => `Hora inválida: Se recibió ${issue.received}`,
+    isoTimeSecond:  (issue) => `Segundo de tiempo inválido: Se recibió ${issue.received}`,
+    isoTimestamp:   (issue) => `Marca de tiempo inválida: Se recibió ${issue.received}`,
+    isoWeek:        (issue) => `Semana inválida: Se recibió ${issue.received}`,
+    length:         (issue) => `Longitud inválida: Se esperaba ${issue.expected} pero se recibió ${issue.received}`,
+    mac:            (issue) => `MAC inválida: Se recibió ${issue.received}`,
+    mac48:          (issue) => `MAC de 48 bits inválida: Se recibió ${issue.received}`,
+    mac64:          (issue) => `MAC de 64 bits inválida: Se recibió ${issue.received}`,
+    maxBytes:       (issue) => `Cantidad de bytes inválidos: Se esperaba máximo ${issue.expected} pero se recibió ${issue.received}`,
+    maxLength:      (issue) => `Longitud inválida: Se esperaba máximo ${issue.expected} pero se recibió ${issue.received}`,
+    maxSize:        (issue) => `Tamaño inválido: Se esperaba máximo ${issue.expected} pero se recibió ${issue.received}`,
+    maxValue:       (issue) => `Valor inválido: Se esperaba máximo ${issue.expected} pero se recibió ${issue.received}`,
+    mimeType:       (issue) => `Tipo de MIME inválido: Se esperaba ${issue.expected} pero se recibió ${issue.received}`,
+    minBytes:       (issue) => `Cantidad de bytes inválidos: Se esperaba mínimo ${issue.expected} pero se recibió ${issue.received}`,
+    minLength:      (issue) => `Longitud inválida: Se esperaba mínimo ${issue.expected} pero se recibió ${issue.received}`,
+    minSize:        (issue) => `Tamaño inválido: Se esperaba mínimo ${issue.expected} pero se recibió ${issue.received}`,
```

Comment:
I added min and max in each of these. But I'm not sure if that should be the case because not even the English version calls this out specifically. Looking forward to your comments.

Reply:
Please remove it as `${issue.expected}` already contains this information.

Reply:
Removed.